src/messages/en.json

```diff
@@ -1,23 +1,34 @@
 {
-    "Index": {
-      "title": "Home",
-      "description": "Testing internationalization."
-    },
-    "Footer": {
-        "pages": {
-            "page1": "HOME",
-            "page2": "ABOUT US",
-            "page3": "SERVICES",
-            "page4": "COMMITEES",
-            "page5": "GET INVOLVED"
+    "Flipping Cards": {
+        "Card1": {
+            "title": "1971 Our Beginning",
+            "content": "In 1971, a group of UC Davis medical students led by Paul Hom held meetings with a group of activist undergraduate students from the California State University of Sacramento to discuss improving health care for elderly Asians and new immigrant families who were struggling to obtain adequate health care due to socioeconomic and language barriers. They decided to start a free clinic to target this problem.",
+            "image": "1971image.png",
```

Comment:
For each of the images, make sure you add `/` at the beginning, so: `/1972image.png`. This will enable you to access the public folder directly.